server/src/main/java/com/genymobile/scrcpy/ScreenEncoder.java

```diff
@@ -155,6 +157,33 @@ private static MediaFormat createFormat(int bitRate, int maxFps, int iFrameInter
             // <https://github.com/Genymobile/scrcpy/issues/488#issuecomment-567321437>
             format.setFloat(KEY_MAX_FPS_TO_ENCODER, maxFps);
         }
+        // Adding additional options specified by the user
+        if(!"-".equals(codecOptions)) {
+            for (String pair : codecOptions.split(",")) {
+                String[] option = pair.split("=");
+                String key = option[0];
+                if(format.containsKey(key)) {
```

Comment:
Oh, contrary to what I said in a previous PR, this does not do what I expected (in fact it just test whether we already set the key, so in practice this is always false).